rts/motoko-rts/src/gc/mark_compact/bitmap.rs

```diff
@@ -100,7 +109,9 @@ impl BitmapIter {
 
             // Inner loop iterates bits in the current word
             while self.current_word != 0 {
-                if self.current_word & 0b1 == 0b1 {
+                if self.current_word & 0b1 == 0b1
+                /* TODO */
```

Comment:
I guess this is what "also below!" above points to? If `!= 0` is really faster (why?) let's fix this as well, the change for it will be smaller than adding the TODO comments.

Reply:
ditto